ouroboros/chainsync/compatibility/types.go

```diff
@@ -39,25 +42,25 @@ func (c *CompatibleResultFindIntersection) UnmarshalJSON(data []byte) error {
 	err = json.Unmarshal(data, &r5)
 	if err != nil {
 		return err
-	} else if r5.IntersectionFound != nil {
-		p := r5.IntersectionFound.Point.ConvertToV6()
-		tip := r5.IntersectionFound.Tip.ConvertToV6()
-		c.Intersection = &p
-		c.Tip = &tip
-		c.Error = nil
-		c.ID = nil
+	} else {
+		r = r5.ConvertToV6()
```

Comment:
It doesn't look like we actually assign this to `c`

Reply:
Good catch.